src/3_branch.js

```diff
@@ -464,7 +466,27 @@ Branch.prototype['setIdentity'] = wrap(callback_params.CALLBACK_ERR_DATA, functi
  */
 /*** +TOC_ITEM #logoutcallback &.logout()& ^ALL ***/
 Branch.prototype['logout'] = wrap(callback_params.CALLBACK_ERR, function(done) {
-	this._api(resources.logout, { }, done);
+	var self = this;
+	this._api(resources.logout, { }, function(err, data) {
+		if (err) { done(err); }
+
+		var data = {
```

Comment:
oops yeah. we should probably actually name this variable something else then though. or just pass it in as a literal into session.set